src/playlist.js

```diff
@@ -305,6 +307,43 @@ const calculateExpiredTime = function(playlist, expiredSync, segmentSync) {
 
     return segmentSync.time - sumDurations(playlist, syncIndex, 0);
   }
+  return 0;
```

Comment:
This should return `null` by default isntead of `0`. If there isn't enough information to determine expired time, returning `0` would be sending false information if there actually is expired time.